modules/magniteAnalyticsAdapter.js

```diff
@@ -633,6 +658,47 @@ const handleBidWon = args => {
   addEventToQueue({ bidsWon: [bidWon] }, bidWon.renderAuctionId, 'bidWon');
 }
 
+window.addEventListener('dmWebVitals', eventObj => {
```

Comment:
How is this event supposed to be launched? Is there a prebid module that I have to load in page? Or, are those events added by the adapter? (It seems no). Or, is there any code that the publisher has to add on his page? which code?

Reply:
This is Demand Manager Specific code see here https://github.rp-core.com/ContainerTag/demand-manager-service/tree/main/bin/lib/dm-web-vitals